Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/validation/core/business/wrapper/Mappings.kt

```diff
@@ -101,8 +101,7 @@ private val DccValidationRule.Type.asExternalType: Type
     get() = when (this) {
         DccValidationRule.Type.ACCEPTANCE -> Type.ACCEPTANCE
         DccValidationRule.Type.INVALIDATION -> Type.INVALIDATION
-        DccValidationRule.Type.BOOSTER_NOTIFICATION ->
-            throw IllegalArgumentException("Not yet supported by dgca verifier")
+        DccValidationRule.Type.BOOSTER_NOTIFICATION -> Type.ACCEPTANCE
```

Comment:
TODO clarify this

Reply:
@BMItr  clarified with Thomas, this part seems to be missing for Android , it is in iOS part 
https://github.com/corona-warn-app/cwa-app-tech-spec/blob/894f9517ee41889bae64f365494e74c60d3f7252/docs/spec/dgc-overview-client.md#ios-invoking-certlogic-for-booster-notifications